neutron/plugins/bigswitch/l3_router_plugin.py

```diff
@@ -94,6 +94,14 @@ def update_router(self, context, router_id, router):
         with context.session.begin(subtransactions=True):
             new_router = super(L3RestProxy,
                                self).update_router(context, router_id, router)
+            if new_router.get(l3.EXTERNAL_GW_INFO):
+                ext_net_id = new_router[l3.EXTERNAL_GW_INFO].get('network_id')
+                ext_net = super(L3RestProxy,
```

Comment:
just use self.get_network here. You only need to call super if you want to call a function in one of the parent classes that you have overridden. 

```python
ext_net = self.get_network(context, ext_net_id)